packages/audio-block/src/types.ts

```diff
@@ -0,0 +1,34 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { AppBridgeBlock, Asset } from '@frontify/app-bridge';
+
+export enum TextPosition {
+    Below = 'Below',
+    Above = 'Above',
+}
+
+export type BlockSettings = {
+    positioning: TextPosition;
+    audio?: Asset;
```

Comment:
```suggestion
```
Shouldn't be part of settings
Please create
```ts
export type BlockAssets = {
	attachments?: Asset[]
	audio?: Asset
}
```
And use it with `useBlockAssets`

Reply:
see the comment before